src/main/java/com/smartcar/sdk/data/BatchResponse.java

```diff
@@ -91,89 +93,125 @@ private static String toCamelCase(String fieldName) {
      * Get response from the /battery endpoint
      *
      * @return the battery status of the vehicle
-     * @throws SmartcarException if the request is unsuccessful
+     *
+     * @throws NoSuchAttributeException if the requested data was not
```

Comment:
```suggestion
     * @throws NoSuchAttributeException if this endpoint was not
```
_do for all of them!_